bundle/default-bundle/src/test/java/io/camunda/connector/runtime/app/CorsConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package io.camunda.connector.runtime.app;
+
+import org.jetbrains.annotations.NotNull;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@Profile("docker-core")
```

Comment:
Maybe we should introduce a "local" profile. `docker-core` could imply we are running in docker.